tools/add_imports/bin/add_imports.dart

```diff
@@ -39,7 +39,7 @@ String createNewSource(String sourceString, Config config) {
       .toList();
 
   return [
-    ...[config.importStringForClass, (e) => 'self.$e = $e']
+    ...[config.importStringForClass, (e) => 'self.$e = $e;']
```

Comment:
I spent like billion years figuring out the problem, and all it was is an optional semi-colon
I'm pretty sure that it was working without it somehow tho 🤷